CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- **AutocompleteInput** Solve double focus issue when navigating through the suggested options with tab and arrow keys.
```

Comment:
Maybe this should go on a ### Fixed title?

Reply:
I agree with @lfp2

Reply:
Also, maybe doesn't need the `Solve` word. It would sound like "Fixed AutocompleteInput double focus..."

Reply:
@lfp2 , @estacioneto thanks for pointing this out, I'll change it!